config/config.ts

```diff
@@ -4,6 +4,7 @@ export class Config {
         : 'dev';
     #accountID = String(process.env.ACCOUNTID);
     #userName = String(process.env.USERNAME);
+    #realityCheckUserName = String(process.env.REALITYCHECKUSERNAME);
```

Comment:
Perhaps we need a reality check password too? Otherwise this code will expect that the reality check user and a normal user to use the same password

Reply:
The password is the same since its generated from the tool. Till something changes, I would prefer not adding that cos I think its a maintenance overhead.

Reply:
This might not be the case in live

Reply:
I wonder if reality check can be messed up in live either. Something to investigate.